packit_service/worker/handlers/copr.py

```diff
@@ -135,8 +137,20 @@ def check_if_actor_can_run_job_and_report(self, actor: str) -> bool:
             return False
         return True
 
+    def get_packit_github_installation_time(self) -> Optional[datetime]:
+        if isinstance(self.project, GithubProject) and (
+            installation := GithubInstallationModel.get_for_github_project(
+                namespace=self.project.namespace, repo_name=self.project.repo
+            )
+        ):
+            return installation.created_at
+        return None
+
     def run(self) -> TaskResults:
-        if self.package_config.srpm_build_deps is not None:
+        installed_at = self.get_packit_github_installation_time()
+        if self.package_config.srpm_build_deps is not None or (
```

Comment:
[srpm_build_deps](https://github.com/packit/packit/blob/4b30b85f3d943ef068d8cdd5d3bf6fbac17e823a/packit/config/common_package_config.py#L147) is a list, so it can be empty, right?
```suggestion
        if self.package_config.srpm_build_deps or (
```

Reply:
Since we use it as an enabler, I would accept an empty string (=non-default value / =user interaction happened ) to build in Copr as well.

Reply:
When thinking about this I realised that I need to transform `None` into an empty list for new installations that do not define `srpm_build_deps` at all...

Reply:
I've just added test cases to cover that.